snap/hooks/install

```diff
@@ -17,15 +17,17 @@ function create_file_structure () {
     # -------------------------------
 
     # Opensearch install folders and set config
-    dir_copy_if_not_exists "bin" "${SNAP_DATA}" 770
-    dir_copy_if_not_exists "jdk" "${SNAP_DATA}" 770
-    dir_copy_if_not_exists "lib" "${SNAP_DATA}" 550
-    dir_copy_if_not_exists "modules" "${SNAP_DATA}" 550
-    dir_copy_if_not_exists "plugins" "${SNAP_DATA}" 770
-
-    dir_copy_if_not_exists "config" "${SNAP_DATA}" 770
-    add_folder "${SNAP_DATA}/config/certificates" 770
-    add_file "${SNAP_DATA}/config/unicast_hosts.txt" 770
+    add_folder "${SNAP_DATA}/etc/opensearch/" 770
+    add_folder "${SNAP_DATA}/usr/share/opensearch/" 770
+    dir_copy_if_not_exists "bin" "${SNAP_DATA}/usr/share/opensearch/" 770
+    dir_copy_if_not_exists "jdk" "${SNAP_DATA}/usr/share/opensearch/" 770
+    dir_copy_if_not_exists "lib" "${SNAP_DATA}/usr/share/opensearch/" 550
+    dir_copy_if_not_exists "modules" "${SNAP_DATA}/usr/share/opensearch/" 550
+    dir_copy_if_not_exists "plugins" "${SNAP_DATA}/usr/share/opensearch/" 770
+    dir_copy_if_not_exists "config" "${SNAP_DATA}/etc/opensearch/" 770
+
+    add_folder "${SNAP_DATA}/etc/opensearch/certificates" 770
+    add_file "${SNAP_DATA}/etc/opensearch/unicast_hosts.txt" 770
 
```

Comment:
We need to add the directory `opensearch_home/extensions --- 770` and file `opensearch_home/extensions/extensions.yml   --- 770` for this work on newer versions (you'll find them in `main`)

Reply:
Thanks, I've added this in.